### ISSUE TITLE
Fix scripts/regen-from-raw so it reports errors on errors

### ISSUE THREAD

**Issue by david-a-wheeler** [metamath/set.mm#1206]
When looking at: https://travis-ci.org/metamath/set.mm/jobs/600097289

It teveals that `scripts/regen-from-raw` is returning a "no error" code when there *are* errors, and we have some files with errors. Details below.

We need to fix the script *and* the problems it's finding.

~~~~
$ scripts/regen-from-raw --force '/HTML' .regened.html

Metamath - Version 0.178 10-Aug-2019          Type HELP for help, EXIT to exit.

MM> Reading source file "set.mm"... 34531089 bytes

34531089 bytes were read into the source buffer.

The source has 161201 statements; 2313 are $a and 33146 are $p.

No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *

if you want to check them.

MM> Continuous scrolling is now in effect.

MM> Reading definitions from $t statement of set.mm...

1446 typesetting statements were read from "set.mm".

Reading "mmcomplex.raw.html"...

Creating "mmcomplex.regened.html"...

MM> Reading "mmnatded.raw.html"...

Creating "mmnatded.regened.html"...

?Warning: The label token "chris/papers/2002-pisa.pdf&quot;&gt;&quot;From"

(referenced in comment of statement "(N/A)") is not a $a or $p statement label.

MM> Reading "mmset.raw.html"...

Creating "mmset.regened.html"...

MM> Reading "mmdeduction.raw.html"...

Creating "mmdeduction.regened.html"...

?Error:  There is a "~" with no label in the comment of statement "(N/A)". 

Check that "`" inside of a math symbol is escaped with "``".

?Warning: The label token "" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)," (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "Q" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "R)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "and" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "and" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)." (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)." (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "(A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

MM> Reading "mmzfcnd.raw.html"...

Creating "mmzfcnd.regened.html"...

MM> Reading "mmnf.raw.html"...

Creating "mmnf.regened.html"...

?Warning: The label token "ax-nin" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-si" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-sn" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-ins2" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-ins3" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-xp" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-typlower" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-cnv" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-1c" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-sset" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

MM> Metamath - Version 0.178 10-Aug-2019          Type HELP for help, EXIT to exit.

MM> Reading source file "iset.mm"... 2825184 bytes

2825184 bytes were read into the source buffer.

The source has 23341 statements; 307 are $a and 6040 are $p.

No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *

if you want to check them.

MM> Continuous scrolling is now in effect.

MM> Reading definitions from $t statement of iset.mm...

231 typesetting statements were read from "iset.mm".

Reading "mmil.raw.html"...

Creating "mmil.regened.html"...

?Warning: The label token "prnaddl" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

The command "scripts/regen-from-raw --force '/HTML' .regened.html" exited with 0.

0.00s$ mkdir -p mpegif-html/

The command "mkdir -p mpegif-html/" exited with 0.

0.01s$ for f in *.regened.html; do mv $f mpegif-html/${f%.regened.html}.html ; done

The command "for f in *.regened.html; do mv $f mpegif-html/${f%.regened.html}.html ; done" exited with 0.

3.66s$ scripts/regen-from-raw --force '/ALT_HTML' .regened.html

Metamath - Version 0.178 10-Aug-2019          Type HELP for help, EXIT to exit.

MM> Reading source file "set.mm"... 34531089 bytes

34531089 bytes were read into the source buffer.

The source has 161201 statements; 2313 are $a and 33146 are $p.

No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *

if you want to check them.

MM> Continuous scrolling is now in effect.

MM> Reading definitions from $t statement of set.mm...

1446 typesetting statements were read from "set.mm".

Reading "mmcomplex.raw.html"...

Creating "mmcomplex.regened.html"...

MM> Reading "mmnatded.raw.html"...

Creating "mmnatded.regened.html"...

?Warning: The label token "chris/papers/2002-pisa.pdf&quot;&gt;&quot;From"

(referenced in comment of statement "(N/A)") is not a $a or $p statement label.

MM> Reading "mmset.raw.html"...

Creating "mmset.regened.html"...

MM> Reading "mmdeduction.raw.html"...

Creating "mmdeduction.regened.html"...

?Error:  There is a "~" with no label in the comment of statement "(N/A)". 

Check that "`" inside of a math symbol is escaped with "``".

?Warning: The label token "" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)," (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "Q" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "R)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "and" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "and" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F))" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "F)))" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)." (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "B" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "A)." (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

?Warning: The label token "(A" (referenced in comment of statement "(N/A)") is

not a $a or $p statement label.

MM> Reading "mmzfcnd.raw.html"...

Creating "mmzfcnd.regened.html"...

MM> Reading "mmnf.raw.html"...

Creating "mmnf.regened.html"...

?Warning: The label token "ax-nin" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-si" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-sn" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-ins2" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-ins3" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-xp" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-typlower" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

?Warning: The label token "ax-cnv" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-1c" (referenced in comment of statement "(N/A)")

is not a $a or $p statement label.

?Warning: The label token "ax-sset" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

MM> Metamath - Version 0.178 10-Aug-2019          Type HELP for help, EXIT to exit.

MM> Reading source file "iset.mm"... 2825184 bytes

2825184 bytes were read into the source buffer.

The source has 23341 statements; 307 are $a and 6040 are $p.

No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *

if you want to check them.

MM> Continuous scrolling is now in effect.

MM> Reading definitions from $t statement of iset.mm...

231 typesetting statements were read from "iset.mm".

Reading "mmil.raw.html"...

Creating "mmil.regened.html"...

?Warning: The label token "prnaddl" (referenced in comment of statement

"(N/A)") is not a $a or $p statement label.

The command "scripts/regen-from-raw --force '/ALT_HTML' .regened.html" exited with 0.
~~~~

**Comment by david-a-wheeler** [comment]
This is fixed by #1211.

**Comment by david-a-wheeler** [comment]
Fixed by #1211, which has been merged, so I'm closing this.
